examples/objective-c/route_guide/ViewControllers.m

```diff
@@ -83,17 +83,24 @@ + (instancetype)noteWithMessage:(NSString *)message
 @interface GetFeatureViewController : UIViewController {
   RTGRouteGuide *service;
 }
+@property (weak, nonatomic) IBOutlet UILabel *outputLabel;
 @end
 
 @implementation GetFeatureViewController
 
 - (void)execRequest {
   void (^handler)(RTGFeature *response, NSError *error) = ^(RTGFeature *response, NSError *error) {
     if (response.name.length) {
+      NSString *str =[NSString stringWithFormat:@"%@\nFound feature called %@ at %@.", self.outputLabel.text, response.location, response.name];
+      self.outputLabel.text = str;
       NSLog(@"Found feature called %@ at %@.", response.name, response.location);
```

Comment:
It looks like we could remove boilerplate for the people reading the example by doing:
```objective-c
if (response.name.length) {
  [self log:@"Found feature called %@ at %@.", response.name, response.location];
} ...
```
The log method can be declared, along with the `outputLabel` and `service` properties, in a superclass that's common to all these view controllers:
```objc
@interface SampleViewController : UIViewController
@property(weak, nonatomic) IBOutlet UILabel *outputLabel;
@property(strong, nonatomic) RTGRouteGuide *service;

- (void)log:(NSString *)formatString, ...;
@end

@implementation SampleViewController
- (void)log:(NSString *)formatString, ... {
  va_list args;
  va_start(args, formatString);
  NSString *str = [NSString stringWithFormat:formatString arguments:args];
  self.outputLabel.text = [self.outputLabel.text stringByAppendingString:str];
  NSLog(str);
  va_end(args);
}
@end
```